client/webpack.config.js

```diff
@@ -56,7 +56,10 @@ module.exports = {
     new webpack.HotModuleReplacementPlugin(),
     new SystemBellPlugin(),
     new webpack.DefinePlugin({
-      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)
+      'process.env': {
+        'NODE_ENV': JSON.stringify(process.env.NODE_ENV),
+        'LUMEN_KEYCLOAK_URL': JSON.stringify(process.env.LUMEN_KEYCLOAK_URL)
```

Comment:
Without it we get syntax error cased by the ":" in the url. My JS skills are not good enough to explain why.